docker/docker-entrypoint.sh

```diff
@@ -47,5 +47,8 @@ pip install -r requirements.txt
 
 wait-for-it.sh -h "${ADDR}" -p "${PORT}" -t 90 -- >&2 echo "Rocket.Chat is ready"
 
+>&2 echo "Checking if hubot is available"
```

Comment:
`...is online`. You use everywhere the term `online`.

Reply:
* Add a new variable `BOT_NAME`. Use it here instead of `hubot`.
* Add the variable to README.